blazor/diagram/annotations/appearance.md

```diff
@@ -538,6 +538,63 @@ N> The default value is [InheritReadOnly](https://help.syncfusion.com/cr/blazor/
 
 Refer to [Constraints](https://blazor.syncfusion.com/documentation/diagram/constraints) to learn about how to enable or disable the annotation constraints.
 
+## Template Support for Annotation
+Diagram provides template support for annotation. you can define a HTML content at tag level and specify the use of template with [UseTemplate](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Diagram.Annotation.html#Syncfusion_Blazor_Diagram_Annotation_UseTemplate) property. If you want to define separate template for each annotation, differentiate the annotaiton by the ID property.
```

Comment:
Diagram provides template support for annotation. You can define HTML content at the tag level and specify the use of a template with [UseTemplate](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Diagram.Annotation.html#Syncfusion_Blazor_Diagram_Annotation_UseTemplate) property. To define a separate template for each annotation, differentiate the annotation by the ID property.